.github/workflows/build-amdvlk-docker.yml

```diff
@@ -13,7 +13,7 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        host-os:        ["ubuntu-20.04"]
+        host-os:        ["ubuntu-22.04"]
         image-template: ["amdvlkadmin/amdvlk_%s%s:nightly"]
```

Comment:
Please fix the previous CI change error first.

And I think you could run the action with your branch to test it first.
And rename the image-template as 'amdvlkadmin/2204/amdvlk_******' during the swtiching.
That will not impact current images.

Reply:
> Please fix the previous CI change error first.

It’s not a big problem that the sanitizer builds are failing right? The other images are uploaded successfully and the sanitizer builds are not run on PRs, so there’s not much impact.

The build of the sanitizer jobs is passing (which was the problem when gpurt was published), it’s just that a regression sneaked in since then. I already fixed that issue in the internal repo a few days ago, I’ll probably port the public part of that PR.

> And I think we should run the actions with your branch to test it first.

I tested the builds with docker locally (for the previous and for this PR), so I think there’s not much that can go wrong?
Even if the base image build fails, it won’t impact PR builds – they will simply use the previous, successful images – and we can revert this PR.

Reply:
The pre-run actions can avoid one more change for llpc docker check, so I suggest that.
If the actions pass, then you could add the llpc docker check codes, and submit them in a PR.

There still has risk for using previous image, that's why we build per 12 hours, right?
I noticed the amdvlk docker build still failed, that will delay your next change about llpc docker check, right?

Reply:
I didn’t understand that.

Anyway, here’s a job running with the changes from this PR: https://github.com/Flakebi/llpc/actions/runs/3647350947